### DISCUSSION TITLE
Write protect part of filesystem?

### DISCUSSION THREAD

**Discussion by teddycool** [davesteele/comitup#239] [Q&A] [answered]
Hi,
I am using comitup and it's awesome, thanx so much for maintaining and sharing this work!
**Background**: I am using a raspberry pi as the base for a bee-hive-camera and want the user to just plug it in, setting up the network with comitup and then configure things from my backend. This works just fine, but..
I am a bit worried to ruin the file-system when just cutting the power to this headless device and would like to safe-gard to always be able to boot with the settings for comitup. For my own settings I can use a ram-disk but for comitup to work I need some persistent memory on the sd-card but would like to write-protect the rest.
**So, my question**: Has anyone some idea or experience how to set this up?

**Comment by davesteele** [comment, marked as answer]
See #8 and linked issues, and particularly [this comment](https://github.com/davesteele/comitup/issues/8#issuecomment-469365778).

Apart from logging, it doesn't write to disk much.

**Reply by teddycool** [reply]
I also found this thread that gave some clues how to make partitions (for a different use-case, hardening the OS):
https://forums.raspberrypi.com/viewtopic.php?t=331031
